.devcontainer/chefs_local/local.json.sample

```diff
@@ -53,8 +53,9 @@
       "port": "8080",
       "rateLimit" : {
         "public": {
-          "windowMs": "900000",
-          "max": "100"
+          "limitApiKey": "120",
+          "limitFrontend": "500",
+          "windowMs": "60000",
```

Comment:
Devs: This is the change that is needed in your `.devcontainers/chefs_local/local.json`.